src/layout/css/_about.scss

```diff
@@ -43,38 +43,38 @@ section p {
   font-size: 15px;
 }
 
-.left-overlay_about_content {
+.ts-gradient-background-start {
   background: linear-gradient(
-    270deg,
-    rgba(17, 191, 86, 0.2) 43%,
-    rgba(5, 173, 73, 0.9) 65%
+    0deg,
+    rgba(17, 191, 86, 0.1) 0%,
+    rgba(5, 173, 73, 0.9) 0%
   );
 }
 
-.right-overlay_about_content {
+.ts-gradient-background-end {
   background: linear-gradient(
     90deg,
-    rgba(17, 191, 86, 0.2) 43%,
-    rgba(5, 173, 73, 0.9) 65%
+    rgba(17, 191, 86, 0.1) 0%,
+    rgba(5, 173, 73, 0.9) 0%
   );
 }
 
-@media (max-width: 750px) {
-  .left-overlay_about_content {
+// Align with Bootstrap's responsive breakpoints, 
+//    https://getbootstrap.com/docs/4.6/layout/overview/#responsive-breakpoints
+@media (min-width: 768px) {
```

Comment:
First, the breakpoint we had previously (750px) is off from the actual breakpoint Bootstrap uses (768px), that's why we saw a glitch for some screen sizes where the text alignment (controlled by Bootstrap/React) did not align with the background gradient (controlled by our CSS rules).

Second, I switched from `max-width`  to `min-width` to be able to use the exact same number as in the breakpoint table. The `min-width` is inclusive, while the `max-width` is exclusive, which means we would have to use `max-width: 767px`.